### PR TITLE
fixed hyperlink

### PR DESCRIPTION
**Pull request by aeongdesu**
- add lang when
Rename register, forgot-password or not?

### CODE REVIEW

File: auth/login.php [hunk 1]
```diff
@@ -108,7 +108,7 @@
     <div class="d-block">
       <label for="password" class="control-label">Password</label>
       <div class="float-right">
-        <!--<a href="auth-forgot-password.html" class="text-small">
+        <!--<a href="password.html" class="text-small">
```
Comment (ilsubyeega) [inline auth/login.php]: currently `forgot_password`

File: auth/login.php [hunk 2]
```diff
@@ -134,7 +134,7 @@
 </div>
 <?if ($session_exists == 0){?>
 <div class="mt-5 text-muted text-center">
-  Don't have an account? <a href="auth-register.html">Create One</a>
+  Don't have an account? <a href="register.html">Create One</a>
```
Comment (ilsubyeega) [inline auth/login.php]: `register` not `register.html`

**Review by ilsubyeega — CHANGES_REQUESTED** [review]

File: auth/login.php (hunk 2 above)
Comment (aeongdesu) [reply auth/login.php]: Ok. I'll fix this.

**Comment by aeongdesu** [comment]
Done. Now you can merge this. Xd

**Review by ilsubyeega — APPROVED** [review]

**Pull request merged**